### DISCUSSION TITLE
Error in 'Route Function' after update last Vike package

### DISCUSSION THREAD

**Discussion by Maikpwwq** [vikejs/vike#1222] [Discussion]
This code for my routes in `index.page.route.js` file used to works fine in my project before I updated `vite-plugin-ssr` to the renamed package `vike`, and I installed [@vite-plugin-vercel/vike](https://github.com/magne4000/vite-plugin-vercel/blob/main/packages/vike-integration/README.md#usage) to work with [vite-plugin-vercel](https://github.com/magne4000/vite-plugin-vercel#usage-with-vike). Please help me to update correctly my [Route Funtion](https://vike.dev/route-function) to handle together routes for `/app/portal-servicios/` and `/app/portal-servicios/@searchInput`.

```
import { resolveRoute } from 'vike/routing'

export default (pageContext) => {
    {
        const result = resolveRoute(
            '/app/portal-servicios/',
            pageContext.urlPathname
        )
        if (result.match) {
            return result
        }
    }

    const result = resolveRoute(
        '/app/portal-servicios/@searchInput',
        pageContext.urlPathname
    )
    return result
}

```

![imagen](https://github.com/vikejs/vike/assets/34326347/711b9b7f-d53e-4799-abc7-76fbaf210087)

**Comment by brillout** [comment]
Create a minimal reproduction, I'll then have a look at it.

**Reply by DaniilIsupov** [reply]
`routeParams` mistakenly empty

**Comment by DaniilIsupov** [comment]
I also have broken `resolveRoute`
Now returns `match: false`
This happened in `0.4.143` version

**Reply by DaniilIsupov** [reply]
in my case, problem was solved by removing extra trailing slash

![image](https://github.com/vikejs/vike/assets/23086234/539e8683-4f7b-4b8d-bd20-4de8b9d98374)
